src/apprt/action.zig

```diff
@@ -341,7 +348,13 @@ pub const Action = union(Key) {
         // For ABI compatibility, we expect that this is our union size.
         // At the time of writing, we don't promise ABI compatibility
         // so we can change this but I want to be aware of it.
-        assert(@sizeOf(CValue) == 16);
+        assert(@sizeOf(CValue) == switch (builtin.target.os.tag) {
+            .windows => switch (builtin.target.cpu.arch) {
+                .x86 => 16,
+                else => 24,
+            },
+            else => 24,
+        });
```

Comment:
This is really sus but I don't know of a better way to handle this.

Reply:
My incredibly crackpot guess would be that this might be because Windows is LLP64 (sizeof(long) == 4) whereas both Linux and macOS are LP64 (sizeof(long) == 8). If this is not it then I legit don't have any more ideas

Reply:
OK, I think that I have a better check now:
```
-        assert(@sizeOf(CValue) == switch (builtin.target.os.tag) {
-            .windows => switch (builtin.target.cpu.arch) {
-                .x86 => 16,
-                else => 24,
-            },
-            else => 24,
+        assert(@sizeOf(CValue) == switch (@sizeOf(usize)) {
+            4 => 16,
+            8 => 24,
+            else => unreachable,
```